src/geo/locate.js

```diff
@@ -1,6 +1,9 @@
 import request from 'superagent'
 import QuickLRU from 'quick-lru'
 import { Agent } from 'http'
+import geojson from './geojson'
+import capitalize from '../capitalize'
+import _ from '../_'
```

Comment:
why are you looking up a directory for a npm module?

Reply:
It's not loading lodash, it's loading `_` from vandelay (which is currently just lodash, but might include monitoring or extensions or something in future)

Reply:
Can you just grab the specific exports you need off of the lodash module? Don't need the layer of indirection with `../_`